src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/BQSR_Dataflow_Transform.java

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.hellbender.dev.pipelines.bqsr;
+
+import com.google.cloud.dataflow.sdk.transforms.PTransform;
+import com.google.cloud.dataflow.sdk.values.PCollection;
+import com.google.cloud.dataflow.sdk.values.PCollectionTuple;
+import htsjdk.samtools.SAMFileHeader;
+import org.broadinstitute.hellbender.dev.tools.walkers.bqsr.BaseRecalibrationArgumentCollection;
+import org.broadinstitute.hellbender.tools.recalibration.RecalibrationTables;
+
+/**
+ *
+ * BaseRecalibration, presented as a PTransform.
+ *
+ * Create the inputs like this:
+ *
+ * PCollection<Read> reads = ...
+ * PCollection<SimpleInterval> knownIntervals = ...
+ * PCollectionTuple inputs =
+ *     PCollectionTuple.of(BQSR_Dataflow.readTag, reads)
+ *                     .and(BQSR_Dataflow.intervalTag, intervals);
+ */
+public class BQSR_Dataflow_Transform extends PTransform<PCollectionTuple, PCollection<RecalibrationTables>> {
+
+    private final SAMFileHeader header;
+    // local or GCS
+    private final String referencePath;
+    private final BaseRecalibrationArgumentCollection toolArgs;
+
+    /**
+     * @param header The SAM header that corresponds to the reads you're going to pass as input.
+     * @param referencePath A path (local or GCS) to the reference file.
+     * @param toolArgs User-provided arguments that configure BaseRecalibration.
+     */
+    public BQSR_Dataflow_Transform(SAMFileHeader header, String referencePath, BaseRecalibrationArgumentCollection toolArgs) {
+        this.header = header;
+        this.referencePath = referencePath;
+        this.toolArgs = toolArgs;
+    }
+
+    @Override
+    public PCollection<RecalibrationTables> apply(PCollectionTuple input) {
+        return BQSR_Dataflow.GetRecalibrationTables(header, input.get(BQSR_Dataflow.readTag), referencePath, toolArgs, input.get(BQSR_Dataflow.intervalTag));
```

Comment:
Does this transform handle aggregation of the tables? Eg., Can we expect that the output `PCollection` will contain a single merged `RecalibrationTables` reflecting all of the data, or will it be a collection of unmerged tables?

Also, how difficult do you think it will be when we eventually refactor this transform to take a `KV` of `Read` paired with overlapping variants/reference bases to fit into the pipeline skeleton? Will it be a minor change or major surgery? (This is beyond the scope of this branch, of course -- I'm just curious so that we know what to expect in the future).